src/core/src/op/reduce_mean.cpp

```diff
@@ -46,6 +46,7 @@ bool evaluate_mean(const HostTensorPtr& arg, const HostTensorPtr& out, const Axi
         NGRAPH_TYPE_CASE(evaluate_mean, u64, arg, out, axes, keep_dims);
         NGRAPH_TYPE_CASE(evaluate_mean, f16, arg, out, axes, keep_dims);
         NGRAPH_TYPE_CASE(evaluate_mean, f32, arg, out, axes, keep_dims);
+        NGRAPH_TYPE_CASE(evaluate_mean, f64, arg, out, axes, keep_dims);
```

Comment:
is it used in CF?

Reply:
I used that in validation purpose. Will remove.

Reply:
Done